cartography/intel/gcp/gke.py

```diff
@@ -28,38 +28,29 @@ def get_gke_clusters(container, project_id):
         res = req.execute()
         return res
     except HttpError as e:
-        reason = compute._get_error_reason(e)
-        if reason == 'invalid':
+        err_as_dict = json.loads(e.content.decode('utf-8'))
```

Comment:
nit: I would do `err = json.loads(e.content.decode('utf-8'))['error'])`
and remove the `status` variable